x-pack/test/alerting_api_integration/basic/tests/actions/builtin_action_types/servicenow.ts

```diff
@@ -9,7 +9,7 @@ import { FtrProviderContext } from '../../../../common/ftr_provider_context';
 import {
   getExternalServiceSimulatorPath,
   ExternalServiceSimulator,
-} from '../../../../common/fixtures/plugins/actions';
+} from '../../../../common/fixtures/plugins/actions_simulators';
```

Comment:
I spent an hour trying to figure out why the `actions` plugin was broken... turned out we have two plugins named `actions`.
This renames the simulator plugin to make this easier to maintain.